documentation/user-interface/main-menu.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: docs
+title: Main Menu
+permalink: /documentation/user-interface/main-menu.html
+---
+# Main Menu
+
+The main menu contains the following items:
+
+* [File](#file)
+* [Edit](#edit)
+* [View](#view)
+* [Help](#help)
+
+## File
+
+The **File** menu provides the following commands:
+
+* **Open Test Directory** - Opens a [test directory](../guides/organize-tests.md#test-directory).
+* **Close Test Directory** - Closes a currently opened test directory.
```

Comment:
Should we say that you will see our Welcome page?

Reply:
AFAIK we have little different menu on Mac, should we mention about it?

Reply:
We shouldn't say about Welcome.

Reply:
>Mac
Ok, I'll see